web/src/emoji_picker.js

```diff
@@ -590,7 +590,11 @@ export function emoji_select_tab($elt) {
     }
     // Handles the corner case of the scrolling back to top.
     if (scrolltop === 0) {
-        currently_selected = section_head_offsets[0].section;
+        if (section_head_offsets.length === 0) {
+            currently_selected = "Popular";
```

Comment:
Please add a comment summarizing how this case is possible, and why we pick this resolution, before this line.